src/agent_IR_RaymondWang_LinYang.py

```diff
@@ -0,0 +1,98 @@
+#trying
+""" Information Ratio """
+import argparse
+import numpy as np
+from collections import deque
+from pedlar.agent import Agent
+import time as tm
+
+
+
+class IRAgent(Agent):
+    name = "IR"
+    def __init__(self, verbose=False, **kwargs):
+        self.verbose=verbose
+        self.slow = deque(maxlen=128)
+        self.fast = deque(maxlen=32)
+        self.last_order = None
+        self.price_list=[]
+        self.std=0
+        self.total_profit_list=[]
+        self.total_profit=0
+        self.order_in_progress=False
+        super().__init__(**kwargs)
+
+
+    # print order when placing an order
```

Comment:
Move the comment to be inside the function using triple quotes """ """

eg
def on_order(self, order):
""" Resets price_list, and updates last_order with current order.id """